src/ChatApp.WebApi/Controllers/ChatController.cs

```diff
@@ -39,9 +41,9 @@ public async Task ProcessStreamingMessage(AIChatRequest request)
             var userInput = request.Messages.Last();
             CreateWriterRequest createWriterRequest = _yamlDeserializer.Deserialize<CreateWriterRequest>(userInput.Content);
 
-            var session = await _creativeWriterApp.CreateSessionAsync(Response);
+            _creativeWriterApp.SetResponseForSession(Response);
```

Comment:
Decoupling additional debug output in chat response message with (AI) Agent creation.

Reply:
Removed additional debug output for now.